corehq/ex-submodules/casexml/apps/phone/data_providers/case/livequery.py

```diff
@@ -113,6 +115,11 @@ def do_livequery(timing_context, restore_state, response, async_task=None):
                 init_progress(async_task, total_cases),
                 total_cases,
             )
+    # Reset loadtest factor for loadtest users after their sync payload
+    # is created.
+    if loadtest_users_enabled(restore_state.domain):
+        for owner_id in owner_ids:
+            reset_loadtest_factor(owner_id)
```

Comment:
This is a code path that is hit very often, do you think that pulling every owner synchronously could pose a performance problem? Otherwise I'd advocate to just kick off a task or something that does this with the list, or do the update in bulk.